specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/afdx.json

```diff
@@ -4373,6 +4373,13 @@
             "$ref": "#/definitions/ActivatedResourceReference"
           }
         },
+        "routes": {
+          "description": "List of routes.",
+          "type": "array",
+          "items": {
+            "$ref": "./cdn.json#/definitions/ResourceReference"
```

Comment:
What type of resource is it which is being referenced?

Reply:
It's our custom definition. You can see it [here](https://github.com/Azure/azure-rest-api-specs/blob/f5895f8357feec0fdcc3bd96105766a45317b8b7/specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/cdn.json#L5012). 

```
    "ResourceReference": {
      "properties": {
        "id": {
          "type": "string",
          "description": "Resource ID."
        }
      },
      "description": "Reference to another resource.",
      "x-ms-azure-resource": true
    },
```